<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbValidateWidget</class>
 <widget class="QWidget" name="DbValidateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="validGroup">
     <property name="title">
      <string>Database Validation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QProgressBar" name="validBar">
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="validLabel">
        <property name="text">
         <string>Validating...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="tableGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Select Table</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QComboBox" name="tableBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="descriptionLabel">
        <property name="text">
         <string>Select the table that holds the names, genders, and answers of all users</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
